pkg/config/appsync_server_config.go

```diff
@@ -41,7 +42,9 @@ type AppSyncConfig struct {
 	RateLimit uint64 `env:"RATE_LIMIT,default=60"`
 
 	// AppSync config
-	AppSyncURL string `env:"APP_SYNC_URL, default=https://www.gstatic.com/exposurenotifications/apps.json"`
+	AppSyncURL    string        `env:"APP_SYNC_URL, default=https://www.gstatic.com/exposurenotifications/apps.json"`
```

Comment:
Is this a sane default? Should it be required and have no default?

Reply:
that's a good point. Left it with no default and kill processing immediately if none is configured.